ding/torch_utils/math_helper.py

```diff
@@ -67,3 +67,27 @@ def cov(
     c = c / fact
 
     return c.squeeze()
+
+
+class OrnsteinUhlenbeckProcess:
```

Comment:
We have implemented OUNoise [here](https://github.com/opendilab/DI-engine/blob/main/ding/rl_utils/exploration.py#L107)